### DISCUSSION TITLE
Can do I make a beatmap in Android?

### DISCUSSION THREAD

**Discussion by KuujaNyeKo** [ppy/osu#19553] [Q&A] [answered]
I can't select an audio or a an image when I want to make a beatmap. 

How do I select an audio? 
![Screenshot_20220802-113001](https://user-images.githubusercontent.com/87144687/182426398-889aa760-54e0-451c-9e08-1f910d1ae3fe.jpg)

**Comment by nagi-desuuu** [comment]
Do your permissions allow osu to access files?

**Reply by KuujaNyeKo** [reply]
Where do I do it from?, I did it from the **application information**, but it remains the same.

**Comment by Theighlin** [comment, marked as answer]
https://github.com/ppy/osu/issues/11911

**Reply by KuujaNyeKo** [reply]
I can't understand lmao

(I am not a developer lol)

**Reply by Theighlin** [reply]
It's a link to an open issue, the title of which is "File selector is broken in Android".
And so this discussion is a duplicate of that, and can be answered as such.
